app/models/optimized_image.rb

```diff
@@ -185,7 +185,7 @@ def self.ensure_safe_paths!(*paths)
     paths.each { |path| raise Discourse::InvalidAccess unless safe_path?(path) }
   end
 
-  IM_DECODERS = /\A(jpe?g|png|ico|gif|webp|avif)\z/i
+  IM_DECODERS = /\A(jpe?g|png|ico|gif|webp|avif|svg)\z/i
```

Comment:
Since you added `svg` in the list, you can simplify [this test](https://github.com/discourse/discourse/blob/efa840f80b866e88598db384de9d70dead8472e0/app/models/optimized_image.rb#L42) as well